repositories.yaml

```diff
@@ -154,18 +154,26 @@ applications:
     url: https://github.com/mozilla/gecko-dev
     notification_emails:
       - chutten@mozilla.com
-    metrics_files:
-      - toolkit/components/glean/metrics.yaml
-    ping_files:
-      - toolkit/components/glean/pings.yaml
     dependencies:
       - glean-core
-      # does not actually depend on glean-android, including for backwards
-      # compat
-      - org.mozilla.components:service-glean
     channels:
       - v1_name: firefox-desktop
         app_id: firefox-desktop
+        metrics_files:
```

Comment:
I am not sure if probe scraper currently supports this concept (metrics files per variant), but it should be straightforward to add if not.